content/docs/command-reference/exp/run.md

```diff
@@ -68,37 +68,29 @@ committing them to the Git repo. Unnecessary ones can be removed with
 
 ## Checkpoints
 
-To track successive steps in a longer <abbr>experiment</abbr>, you can register
-checkpoints with DVC during your code or script runtime (similar to a logger).
+To track successive steps in a longer or deeper <abbr>experiment</abbr>, you can
+register checkpoints from your code (similar to a logger). Each `dvc exp run`
+will resume from the last checkpoint.
 
-To do so, first mark stage `outs` with `checkpoint: true` in `dvc.yaml`. At
-least one checkpoint <abbr>output</abbr> is needed so that the experiment can
-later continue from that output's last cached state.
+First, mark at least stage <abbr>output</abbr> with `checkpoint: true` in
+`dvc.yaml`. This is needed so that the experiment can resume later, based on the
+<abbr>cached</abbr> output(s) (circular dependency).
```

Comment:
We don't mention circular dependencies anywhere else in this doc, this seems like it might be confusing to include it here

Reply:
We don't mention that concept in general (except once in https://dvc.org/doc/api-reference/make_checkpoint). I just decided to stop avoiding it in this PR since it's an important one for checkpoints, I think. Keep in mind this should (eventually will) be moved to a proper guide where we can expand a bit more on this. So I think it's OK to keep it.